prac_08/unreliable_car.py

```diff
@@ -0,0 +1,14 @@
+from prac_08.car import Car
+from random import randint
+
+class UnreliableCar(Car):
+    def __init__(self, name, fuel, reliability):
+        super().__init__(name, fuel)
+        self.reliability = reliability
+
+    def drive(self, distance):
+        random_number = randint(1, 100)
+        if random_number >= self.reliability:
+            distance = 0
+        drive_dist = super().drive(distance)
```

Comment:
Don't abbreviate variable names. Why is `drive_distance` or `distance_driven` so hard to type?